.travis.yml

```diff
@@ -34,10 +34,7 @@ jobs:
 
 addons:
   apt:
-    sources:
```

Comment:
I removed `llvm-toolchain-xenial-6.0` and `clang-6.0`. I think we don't need it anymore. Is it ok?